### PR TITLE
[Docs] Fix link to wontfix label

### PR DESCRIPTION
**Pull request by SaschaMann**

### CODE REVIEW

File: docs/maintainers/repository-structure.md [hunk 1]
```diff
@@ -112,6 +112,7 @@ You can combine labels to find the issues you'd like, e.g. to [list all the issu
 [github-issues-status-help-wanted]: https://github.com/exercism/v3/issues?q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted
 [github-issues-status-in-progress]: https://github.com/exercism/v3/issues?q=is%3Aopen+is%3Aissue+label%3Astatus%2Fin-progress
 [github-issues-status-draft]: https://github.com/exercism/v3/issues?q=is%3Aopen+is%3Aissue+label%3Astatus%2Fdraft
+[github-issues-status-wontfix]: https://github.com/exercism/v3/issues?q=is%3Aissue+label%3Astatus%2Fwontfix+is%3Aclosed
```
Comment (iHiD) [inline docs/maintainers/repository-structure.md:115]: Why do we need this (ie what's wrong with the one on the line below)

Comment (SaschaMann) [reply docs/maintainers/repository-structure.md:115]: If you view the file in the current state, the link already exists, it's just not linking to anywhere: 
grafik

Comment (iHiD) [reply docs/maintainers/repository-structure.md:115]: Should the LOC below this be removed then?

Comment (SaschaMann) [reply docs/maintainers/repository-structure.md:115]: I completely missed that. oops.

File: docs/maintainers/repository-structure.md [hunk 2]
```diff
@@ -112,6 +112,7 @@ You can combine labels to find the issues you'd like, e.g. to [list all the issu
...
 [status/wontfix]: https://github.com/exercism/v3/issues?q=is%3Aclosed+is%3Aissue+label%3Astatus%2Fwontfix
```
Comment (SaschaMann) [inline docs/maintainers/repository-structure.md:116]:
```suggestion
```

File: docs/maintainers/repository-structure.md (hunk 1 above)
Comment (SaschaMann) [reply docs/maintainers/repository-structure.md:115]: fixed

Comment (iHiD) [reply docs/maintainers/repository-structure.md:115]: Sweet! :)

**Review by iHiD — APPROVED** [review]

**Pull request merged**